frontend/src/constants/assets.js

```diff
@@ -17,6 +17,7 @@ export { default as BRAND_PENCIL } from "../assets/images/BrandPencil.svg";
 export { default as BRAND_DOCUMENT } from "../assets/images/BrandDocument.svg";
 export { default as RED_CLOCK } from "../assets/images/late-clock-red.svg";
 export { default as EDIT } from "../assets/images/Edit.svg";
+export { default as EDITOUTLINE } from "../assets/images/EditOutline.svg";
```

Comment:
For consistency, should be in SCREAMING_SNAKE_CASE

Reply:
Yes, updated.